code/weapon/weapons.cpp

```diff
@@ -2557,22 +2557,22 @@ int parse_weapon(int subtype, bool replace, const char *filename)
 		}
 
 		if (optional_string("+Failure Launch Delay Multiplier for AI:")) {
-			int delay_multiplier;
-			stuff_int(&delay_multiplier);
+			float delay_multiplier;
+			stuff_float(&delay_multiplier);
 			if (delay_multiplier > 0) {
 				wip->cmeasure_failure_delay_multiplier_ai = delay_multiplier;
 			} else {
-				Warning(LOCATION,"\"+Failure Launch Delay Multiplier for AI:\" should be >= 0 (read %i). Value will not be used. ", delay_multiplier);
+				Warning(LOCATION,"\"+Failure Launch Delay Multiplier for AI:\" should be >= 0 (read %f). Value will not be used. ", delay_multiplier);
```

Comment:
The error message should say `> 0`, not `>= 0`, since that's what the condition actually checks